src/player.js

```diff
@@ -871,11 +875,11 @@ class Player {
      * @returns {boolean} Success
      */
     setToStartingHero(heroClass = this.heroClass) {
-        let hero_card = heroClass + " Starting Hero";
-        hero_card = game.functions.getCardByName(hero_card);
+        let heroCardName = heroClass + " Starting Hero";
+        let heroCard = game.functions.getCardByName(heroCardName);
```

Comment:
Maybe rename all variables to use camel case instead of snake case. Also maybe rename all single character variables names to be more descriptive.

Reply:
#257 #258